controller/src/main/java/io/pravega/controller/store/stream/ZKStoreHelper.java

```diff
@@ -259,18 +269,27 @@ public ZKStoreHelper(final CuratorFramework cf, Executor executor) {
     }
 
     CompletableFuture<Void> createZNodeIfNotExist(final String path) {
+        return createZNodeIfNotExist(path, true);
+    }
+
+    CompletableFuture<Void> createZNodeIfNotExist(final String path, final boolean createParent) {
```

Comment:
This method seems to be duplicate of the method above, but without the data parameter. Why not use the above method with `new byte[0]` as the value of data param. We have used that trick in ZKCheckpointStore.